### ISSUE TITLE
image_to_string one character predict two result

### ISSUE THREAD

**Issue by er778899789** [madmaze/pytesseract#330]
1. I put a image like below to predict the character. But sometimes, I will get the two result.
How can I get only the best accuracy result?
![ROI_6](https://user-images.githubusercontent.com/32409355/109593298-a4a56b00-7b4b-11eb-941c-47f6c6962413.png)
the result I get : 83

2. I set the config to pytesseract. I want the predict result only from the list in config. But when I put the full width character 0 (like below image) I think it predict the image to english character 'O', and it will show nothing to result. So I change the '0' to 'O', and the result will show up. And the issue sometimes show up not only in 'O' but '6' to 'G'. If I don't set 'G' to config, and the result will show nothing either.
How can I get the predict result only from the list I create?
![ROI_4](https://user-images.githubusercontent.com/32409355/109593685-652b4e80-7b4c-11eb-9659-ec3cff4971a7.png)
![擷取1](https://user-images.githubusercontent.com/32409355/109593711-707e7a00-7b4c-11eb-879e-7c479ca46d2a.PNG)

**Comment by int3l** [comment]
Please take a look at the following issues: [issues with whitelist](https://github.com/madmaze/pytesseract/issues?q=is%3Aissue+is%3Aclosed+whitelist)
TLDR: It depends what version of tesseract you have and the result might be different if you decide to convert the image to `PIL.Image` object.

**Comment by er778899789** [comment]
@int3l 
Hey, thanks for answer the question.
I want to ask which tesseract version can slove my problem like first comment ? 
I don't get any informations on google about the questions. 
Or I need to edit the code in pytesseract site-package by myself ?

**Comment by int3l** [comment]
I am not familiar which specific versions support this feature.
According to [this issue](https://github.com/madmaze/pytesseract/issues/179), they report that 4.1.0 work, but I can't confirm.
